Kernel/Graphics/Intel/DisplayConnectorGroup.cpp

```diff
@@ -147,12 +147,14 @@ ErrorOr<void> IntelDisplayConnectorGroup::initialize_gen4_connectors()
     // Note: Just assume we will need one Gen4 "transcoder"
     // Note: Main block of registers starting at HorizontalTotalA register (0x60000)
     auto transcoder_registers_paddr = m_mmio_first_region.pci_bar_paddr.offset(0x60000);
+    // Note: Main block of Pipe registers starting at PipeA_DSL register (0x70000)
+    auto pipe_registers_paddr = m_mmio_first_region.pci_bar_paddr.offset(0x70000);
```

Comment:
Is there a reason we can't squash ` Kernel/IntelGraphics: Move pipe management to the Transcoder class ` into the commit that creates CRTDisplayTranscoder?